core/admin/Dockerfile

```diff
@@ -1,61 +1,31 @@
-# First stage to build assets
-ARG DISTRO=alpine:3.14.5
+# syntax=docker/dockerfile-upstream:1.4.3
 
-FROM node:16-alpine3.16 as assets
-
-COPY package.json ./
-RUN set -eu \
- && npm config set update-notifier false \
- && npm install --no-fund
-
-COPY webpack.config.js ./
-COPY assets ./assets
-RUN set -eu \
- && sed -i 's/#007bff/#55a5d9/' node_modules/admin-lte/build/scss/_bootstrap-variables.scss \
- && for l in ca da de:de-DE en:en-GB es:es-ES eu fr:fr-FR he hu is it:it-IT ja nb_NO:no-NB nl:nl-NL pl pt:pt-PT ru sv:sv-SE zh; do \
-      cp node_modules/datatables.net-plugins/i18n/${l#*:}.json assets/${l%:*}.json; \
-    done \
- && node_modules/.bin/webpack-cli --color
-
-
-# Actual application
-FROM $DISTRO
-ARG VERSION
-
-ENV TZ Etc/UTC
+# admin image
+FROM base
 
+ARG VERSION=local
 LABEL version=$VERSION
 
-# python3 shared with most images
-RUN set -eu \
- && apk add --no-cache python3 py3-pip py3-wheel git bash tzdata \
- && pip3 install --upgrade pip
+RUN set -euxo pipefail \
+  ; apk add --no-cache libressl mariadb-connector-c postgresql-libs
 
-RUN mkdir -p /app
-WORKDIR /app
+COPY mailu/ ./mailu/
+RUN set -euxo pipefail \
+  ; venv/bin/pybabel compile -d mailu/translations
```

Comment:
I'd move this further down the file... just before the RUN/VERSION so that more caching layers can be reused